src/vs/workbench/contrib/interactiveEditor/browser/interactiveEditorWidget.ts

```diff
@@ -496,9 +496,13 @@ export class InteractiveEditorWidget {
 		this._elements.message.style.webkitLineClamp = expansionState === ExpansionState.NOT_CROPPED ? 'none' : (expansionState === ExpansionState.EXPANDED ? MESSAGE_EXPANDED_NUMBER_LINES.toString() : MESSAGE_CROPPED_NUMBER_LINES.toString());
 	}
 
-	updateInfo(message: string): void {
+	updateInfo(message: string | Array<HTMLSpanElement | string>): void {
```

Comment:
might be nicer to leave the signature be just `message: string` and use `renderLabelWithIcons` inside of it